runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -35,21 +35,14 @@
 " The communication with gdb uses GDB/MI.  See:
 " https://sourceware.org/gdb/current/onlinedocs/gdb/GDB_002fMI.html
 "
-" For neovim compatibility, the vim specific calls were replaced with neovim
-" specific calls:
-"   term_start -> termopen
-"   term_sendkeys -> chansend
-"   term_getline -> getbufline
-"   job_info && term_getjob -> using linux command ps to get the tty
-"   balloon -> nvim floating window
+" NEOVIM COMPATIBILITY
 "
-" The code for opening the floating window was taken from the beautiful
-" implementation of LanguageClient-Neovim:
-" https://github.com/autozimu/LanguageClient-neovim/blob/0ed9b69dca49c415390a8317b19149f97ae093fa/autoload/LanguageClient.vim#L304
-"
-" Neovim terminal also works seamlessly on windows, which is why the ability
-" Author: Bram Moolenaar
-" Copyright: Vim license applies, see ":help license"
+" The vim specific functionalities were replaced with neovim specific calls:
+" - term_start -> termopen
+" - term_sendkeys -> chansend
+" - term_getline -> getbufline
+" - job_info && term_getjob -> using linux command ps to get the tty
+" - balloon -> vim.lsp.util.open_floating_preview
```

Comment:
I think the point here is as a first guide for unsuspecting goobers trying to port a runtime file update, so this list might still be useful information?

Reply:
Might still be useful then. Could do with some sprucing up though, it hasn't been touched much since the original port. (also is that `ps` comment even correct? :thinking:)

Reply:
Yeah, if it's outdated, it's better to remove than to keep it in (the `balloon` one looks sus).

Reply:
`balloon` comment is from this PR, as Vim uses :balloon: tooltips for showing eval when hovering the mouse over a variable (similar to a floating window in that sense that it pops out, I suppose).

Reply:
The ps thing was indeed outdated (since the initial nvim support was added :grin:), and `nvim_get_chan_info` is used instead. Otherwise looks fine, though I haven't really tried to fully understand the implementation.

Reply:
Thanks! I think the list looks fine now.

I also see `uv.kill` replacing `job_start` for sending SIGINT, but I'll have to do a more thorough review later to see if there's more differences.